tests/letstest/requirements.txt

```diff
@@ -4,7 +4,7 @@ boto3==1.14.11
 botocore==1.17.11
 cffi==1.14.0
 colorama==0.4.3
-cryptography==2.8
+cryptography==3.1
```

Comment:
Do we need specific capabilities from cryptography 3.x ?

Reply:
Probably no. I think it was testing around. I've reverted it.